app/locale.js

```diff
@@ -270,9 +270,24 @@ const defaultLocale = function () {
 // Initialize translations for a language providing an optional
 // callback executed after the translation caching process
 // is complete.
-exports.init = function (language, cb) {
-  // Default to noop callback
-  cb = cb || function () {}
+exports.init = function (language) {
+  // If this is in the main process
+  if (ipcMain) {
+    // Respond to requests for translations from the renderer process
+    ipcMain.on('translations', function (event, arg) {
+      // Return the entire set of translations synchronously
+      event.returnValue = translations
+    })
+
+    // TODO: There shouldn't need to be a REQUEST_LANGUAGE event at all
+    // Respond to requests for the currently configured language code
+    ipcMain.on(REQUEST_LANGUAGE, function (event) {
```

Comment:
wait- just noticed this ipc handling code was already in here LOL- but yeah, I think the locale code that was in index.js was unnecessary